frontend/simple/views/CreateGroup.vue

```diff
@@ -4,13 +4,14 @@
     <div class="section columns is-centered">
       <div class="column is-two-thirds">
         <transition name="fade" mode="out-in">
-          <router-view :group="form" :v="$v.form" @input="payload => updateGroupData(payload)">
+          <router-view :group="form" :v="$v.form" v-on:next="next" v-on:focusnext="focusNext" v-on:focusfinish="focusFinish" v-on:finish="submit" @input="payload => updateGroupData(payload)">
```

Comment:
Recommend using `@` shortcut for `v-on` to maintain consistency and cause it's pretty

Reply:
Also recommend delete `next` event and rename `focusnext` to `focusref` (or `focusRef`) for clarity.

Then, to handle the different buttons refs, pass in the button ref name as the event parameter.